.flowconfig

```diff
@@ -23,7 +23,7 @@
 .*/Libraries/polyfills/.*
 
 ; Ignore metro
-.*/node_modules/metro/.*
+; .*/node_modules/metro/.*
```

Comment:
This needs to be kept in sync with the .flowconfig.android file and will fail to land if they are not in sync.

Reply:
Good spot - pushed a commit to update android. Thanks